src/utils/convertToSVG.ts

```diff
@@ -11,27 +11,10 @@ const setAttributes = (attrs) =>
         .map(([key, val]) => `${camelCaseToKebabCase(key)}="${val}"`)
         .join(" ");
 
-export const convertToSVG = ({ icon }: IconSetItem, size = 32): string => {
+export const convertToSVG = ({ icon }: IconSetItem, size = 32, isFile = false): string => {
   const scaledIcon = scaleIcon({ icon, properties: { name: "" } }, size / 1024);
 
-  const svg = `<svg viewbox="0 0 ${scaledIcon.icon.width} ${scaledIcon.icon.width}" width="${scaledIcon.icon.width}" height="${scaledIcon.icon.width}" stroke="currentColor" fill="currentColor">{{paths}}</svg>`;
-
-  const paths = scaledIcon.icon.paths
-    .map((path, index) =>
-      `<path d="${path}" {{attrs}} />`.replace(
-        "{{attrs}}",
-        scaledIcon.icon.attrs ? setAttributes(scaledIcon.icon.attrs[index]) : ""
-      )
-    )
-    .join("");
-
-  return svg.replace("{{paths}}", paths);
-};
-
-export const convertToSVGDocument = ({ icon }: IconSetItem, size = 32): string => {
-  const scaledIcon = scaleIcon({ icon, properties: { name: "" } }, size / 1024);
-
-  const svg = `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewbox="0 0 ${scaledIcon.icon.width} ${scaledIcon.icon.width}" width="${scaledIcon.icon.width}" height="${scaledIcon.icon.width}" stroke="currentColor" fill="currentColor">{{paths}}</svg>`;
+  const svg = `<svg ${isFile && `xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"`} viewbox="0 0 ${scaledIcon.icon.width} ${scaledIcon.icon.width}" width="${scaledIcon.icon.width}" height="${scaledIcon.icon.width}" stroke="currentColor" fill="currentColor">{{paths}}</svg>`;
```

Comment:
Great, but there is a little bug.

If `isFile` is equal to `false`, SVG appear like this.

```js
'<svg false viewbox="0 0 32 32">'
```

It would help if you used a variable for this

```js
const fileAttr = !isFile ?  '' : 'xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"';
const svg = `<svg ${fileAttr} ... ...`
````